addons/marketing_card_event/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': 'Marketing Card For Events',
+    'version': '1.1',
+    'category': 'Marketing/Marketing Card',
+    'summary': 'Generate dynamic shareable cards for event attendees and sponsors.',
```

Comment:
A bit strange since we would need to install those as well, but that's part of the heuristics. Not sure if there is a better wording :shrug:

Reply:
Yup, sounds nice :)